ui/app/components/dashboard/quick-actions-card.js

```diff
@@ -66,6 +72,9 @@ export default class DashboardQuickActionsCard extends Component {
           buttonText: 'Issue leaf certificate',
           model: 'pki/role',
           route: 'vault.cluster.secrets.backend.pki.roles.role.generate',
+          nameKey: 'id',
+          queryObject: { backend: this.selectedEngine.id },
+          objectKeys: ['id'],
```

Comment:
just to note - `objectKeys` returns `id` already, so we can remove this if we don't need any additional model attributes returned